src/promptflow/promptflow/_sdk/_service/templates/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
```

Comment:
we can public expose the promptflow.start_trace(), when we have real ui

Reply:
how about make this a separate PR? I think we need better CHANGELOG with this public

Reply:
We will expose the function first (so we can change sample to expected function) and anounce in CHANGELOG later.

Reply:
as offline synced, exposed but not announce